rviz_visual_testing_framework/src/visual_test_fixture.cpp

```diff
@@ -56,13 +56,17 @@ void VisualTestFixture::SetUpTestCase()
   std::string package_share_directory = ament_index_cpp::get_package_share_directory(
     "rviz_visual_testing_framework");
   if (VisualTest::generateReferenceImages()) {
-    visualizer_app_->loadConfig(QDir::toNativeSeparators(
-        QString::fromStdString(package_share_directory +
-        "/config/visual_tests_default_config.rviz")));
+    visualizer_app_->loadConfig(
+      QDir::toNativeSeparators(
+        QString::fromStdString(
+          package_share_directory +
+          "/config/visual_tests_default_config.rviz")));
   } else {
-    visualizer_app_->loadConfig(QDir::toNativeSeparators(
-        QString::fromStdString(package_share_directory +
-        "/config/visual_tests_test_image_config.rviz")));
+    visualizer_app_->loadConfig(
+      QDir::toNativeSeparators(
+        QString::fromStdString(
+          package_share_directory +
+          "/config/visual_tests_test_image_config.rviz")));
```

Comment:
This is another instance where I think the rule is too aggressive. I think this is what should have been produced:

```c++
    visualizer_app_->loadConfig(QDir::toNativeSeparators(
      QString::fromStdString(
        package_share_directory + "/config/visual_tests_test_image_config.rviz")));
```

Actually this is probably best:

```c++
    visualizer_app_->loadConfig(QDir::toNativeSeparators(QString::fromStdString(
      package_share_directory + "/config/visual_tests_test_image_config.rviz")));
```

but given the original code, the first is reasonable for the linter to produce.

Basically it should be greedy, trying to get as many `(` on the first line so long as the matching `)` are not followed by additional arguments. I would, again, guess that the linter cannot handle such logic. So I guess it's more of a point in favor of maybe leaving this setting unset rather than deal with the "false positives" or "over matching" issues.